src/ast/ddl.rs

```diff
@@ -3029,7 +3029,9 @@ pub struct CreateTrigger {
     /// FOR EACH ROW
     /// EXECUTE FUNCTION trigger_function();
     /// ```
-    pub period: TriggerPeriod,
+    ///
+    /// This may be omitted in SQLite, the effect is equivalent to BEFORE.
```

Comment:
```suggestion
```
we can skip the description since the optionality is clear from the representation (and the conditions can change over time)